README.md

```diff
@@ -8,28 +8,38 @@
 
 Data contracts bring data providers and data consumers together.
 
-A _data contract_ is a document that defines the structure, format, semantics, quality, and terms of use for exchanging data between a data provider and their consumers. 
-Think of an API, but for data. 
-A data contract is implemented by a data product or other data technologies, even legacy data warehouses. 
-Data contracts can also be used for the input port to specify the expectations of data dependencies and verify given guarantees.
-
-The _data contract specification_ defines a YAML format to describe attributes of provided data sets. 
-It is data platform neutral and can be used with any data platform, such as AWS S3, Google BigQuery, Azure, Databricks, and Snowflake. 
-The data contract specification is an open initiative to define a common data contract format. 
+A _data contract_ is a document that defines the structure, format, semantics, quality, and terms of use for exchanging
```

Comment:
Too many changes in the PR (auto format?). Please keep the PR focused on the functional change.

Reply:
Done roll-back on auto-complete.